atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -92,42 +104,40 @@ static void waitForSchemaVersions(Cassandra.Client client, String tableName, int
                 .append(" and examine their logs to determine the issue.")
                 .append(" Fixing the underlying issue and restarting Cassandra")
                 .append(" should resolve the problem. You can quick-check this with 'nodetool describecluster'.");
-        throw new IllegalStateException(sb.toString());
+
+        // 1 real schema version that is correctly distributed + precisely one node that is at version 'unreachable'
```

Comment:
Comments above complex booleans often imply that the clause deserves its own method, but this is a marginal call here. `if (allowUnresponsiveNode && exactlyOneNodeIsUnreachableAndOthersAgree(versions))`?

Related nit, not actionable in this PR: this method does two things:
1. Fetches schema versions
2. Handles the case where we don't get complete agreement

It could be two methods - and possibly a different object, SchemaVersionChecker, that would allow us to define its contract with unit tests. @jboreiko for SA/comment.